From 804d56e0a996535cb12f114fc17c53b346c78215 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@st.com>
Date: Wed, 20 Apr 2022 10:55:09 +0200
Subject: [PATCH 0360/1695] dts: stm32: add support for RTC on stm32mp251.dtsi

Add RTC node under SoC node in stm32mp251.dtsi.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 86bd1921fc813eeaacd7fea3d52807486eba77c4 dts: stm32: add support for RTC on stm32mp251.dtsi]
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
Change-Id: I2fd492acda4a01286eb1b66054deb45369590a58
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/248718
Reviewed-by: Valentin CARON <valentin.caron@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349072
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp251.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index 0f4139851..765ff7ac5 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -587,7 +587,11 @@
 		};
 
 		rtc: rtc@46000000 {
+			compatible = "st,stm32mp25-rtc";
 			reg = <0x46000000 0x400>;
+			clocks = <&rcc CK_BUS_RTC>, <&rcc RTC_CK>;
+			clock-names = "pclk", "rtc_ck";
+			interrupts = <GIC_SPI 15 IRQ_TYPE_LEVEL_HIGH>;
 			status = "disabled";
 		};
 
-- 
2.47.2

